LocalSettings.php

```diff
@@ -560,14 +560,14 @@
 			
 			// Decline reasons:
 			'Needs more details' => 'Can you give us a few more details on the purpose for, scope of, and topic of your wiki, and briefly describe some of your wiki\'s content in approximately 2-3 sentences? Additionally can you elaborate on your wiki\'s scope and topical focus a bit further? A few sentences describing the scope of your wiki and the sort of content it will contain should be helpful. Please go back into your original request and add to, but do not replace, your existing description. Thank you.',
-			'Needs a better subdomain' => 'The scope and purpose of the wiki sound good enough. However, your requested subdomain is not very obvious. Please change it to something that better reflects your wiki\'s purpose. Thank you.',
+			'Invalid subdomain' => 'The scope and purpose of the wiki seem clear enough. However, your requested subdomain is either invalid, is too generic, or suggests the wiki is an English language or multilingual wiki when it is not. Please change it to something that better reflects your wiki\'s purpose and scope. Thank you.',
```

Comment:
It is pretty common for people to include "miraheze" in their subdomain which might indicate that they are affiliated with MH in some way. If we are mentioning the common issues with subdomains then it should be included, too.

Reply:
@dmehus ^^

Reply:
> @dmehus ^^

@R4356th What are you proposing? I mean, we can add an additional reason to the list of three possible reasons, though we don't want to add _too_ many. I think the "Invalid subdomain" short description should stay as is, though, for concision, though we could possibly amend it to "Invalid or unclear subdomain"?

Reply:
Should be done now as well.

Reply:
Thank you. 😃